components/widgets/modals/full_screen_modal.tsx

```diff
@@ -47,9 +65,19 @@ export default class FullScreenModal extends React.Component<Props> {
                 appear={true}
             >
                 <div className='FullScreenModal'>
+                    {this.props.onGoBack &&
+                        <BackIcon
+                            tabIndex='0'
```

Comment:
If the icons are wrapped in `<button>` tags, the `tabIndex` attribute should be removed from the icon `<span>` tags.